include/netfilter.mk

```diff
@@ -152,7 +152,7 @@ $(eval $(if $(NF_KMOD),$(call nf_add,NF_REJECT6,CONFIG_NF_REJECT_IPV6, $(P_V6)nf
 
 $(eval $(if $(NF_KMOD),$(call nf_add,NF_IPT6,CONFIG_IP6_NF_IPTABLES, $(P_V6)ip6_tables),))
 
-$(eval $(if $(NF_KMOD),$(call nf_add,NF_CONNTRACK,CONFIG_NF_DEFRAG_IPV6, $(P_V6)nf_defrag_ipv6),))
+$(eval $(if $(NF_KMOD),$(call nf_add,NF_DEFRAG,CONFIG_NF_DEFRAG_IPV6, $(P_V6)nf_defrag_ipv6),))
```

Comment:
Here I believe you forgot to add the "6"

Reply:
Actually it is not the case, at first I correctly put nf_defrag_ipv4 into nf-defrag package and nf_defrag_ipv6 into nf-defrag6 package, then let nf_conntrack depend on nf-defrag and nf-defrag6, and as a result many other packages reported the dependency error. So I chose to make the same mistake as nf-conntrack and put nf_defrag_ipv6 into the nf-defrag package, while nf-defrag6 is also kept empty with nf-conntrack6.

> > I have a question about IPv6, originally nf-conntrack packet contains nf_defrag and nf_defrag6, but nf-conntrack6 packet is empty, why is that?

> That is indeed a mistake, that must be corrected.

Of course I only just learned that nf-conntrack is a mistake.

I think I need to fix the nf-conntrack bug together to make it work.

Reply:
Yeah, please give it a try, if you can't fix the dependency issue just do as conntrack and later I'll correct it if necessary.